packages/common/src/ui/layout/tables/components/Header/Header.test.tsx

```diff
@@ -92,10 +91,12 @@ describe('HeaderRow', () => {
       </TestingProvider>
     );
 
-    const idHeader = getByRole('button', { name: /name/i });
+    const nameHeader = getByRole('button', { name: /name/i });
 
-    act(() => userEvent.click(idHeader));
+    userEvent.click(nameHeader);
 
-    expect(onSortBy).toBeCalledWith(expect.objectContaining({ key: 'name' }));
+    waitFor(() => {
+      expect(onSortBy).toBeCalledWith(expect.objectContaining({ key: 'name' }));
+    });
```

Comment:
this was failing because I added debounce to the headers and didn't update the test.